### DISCUSSION TITLE
JK PB Series Manuals

### DISCUSSION THREAD

**Discussion by TheSmartGerman** [txubelaxu/esphome-jk-bms#17] [General]
https://www.akkudoktor.net/forum/datenblaetter-manuals-usw/manual-jk-pb-serie/

There are some new manuals

Have a lock at the panel board. left dual connectors RS485 and CAN are swaped:
![image](https://github.com/user-attachments/assets/371151f9-0889-4adc-855f-e9021faf6fdd)

This is the right order, older picures even on JK website show it the other way round.

**Comment by txubelaxu** [comment]
Many Thanks. I have just updated the info with this picture.
